core/src/test/java/org/apache/iceberg/TestUpdateRequirements.java

```diff
@@ -199,7 +199,21 @@ public void upgradeFormatVersion() {
   }
 
   @Test
-  public void upgradeFormatVersionForView() {
+  public void upgradeFormatVersionV3() {
```

Comment:
so technically you don't need to add a new test. I'd probably just slightly adjust `upgradeFormatVersion()` and make it a parameterized test:
```
@ParameterizedTest
@ValueSource(ints = {2, 3}
public void upgradeFormatVersionTo(int version) {...}
```

I'd then do the same pattern for `upgradeFormatVersionForView()`:

```
@ParameterizedTest
@ValueSource(ints = {2, 3}
public void upgradeFormatVersionForViewTo(int version) {...}
```

Reply:
Thanks updating this with parameterization